app/AttributeTypes/BooleanAttribute.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\AttributeTypes;
+
+class BooleanAttribute extends AttributeBase
+{
+    protected static string $type = "boolean";
+    protected static bool $inTable = true;
+    protected static ?string $field = 'int_val';
+
+    public static function fromImport(string $data) : mixed {
+        return 
+                $data == 1 || $data == '1' ||
```

Comment:
Can $data be 1? when it's always a text it will be covered by $data=='1' and inval($data)>0

Reply:
$data == '1' is also covered with intval($data) > 0

Reply:
Good point. `1` should definitely be allowed in csv. Thus I'll change type from `string` to `int|float|bool|string`

Reply:
Totally agree.